FrontEnd/src/Components/ImageCarousel.jsx

```diff
@@ -1,8 +1,8 @@
 import { Box, Image, HStack, Text, Flex } from "@chakra-ui/react";
 import { useState } from "react";
-import slides from "./CardData/Slides";
+import PropTypes from "prop-types";
 
-const ImageCarousel = () => {
+const ImageCarousel = ({ imgurl }) => {
```

Comment:
It seems a array of URLs is being pass as a prop to the component, the naming of the prop can be improved

Reply:
Sure, will make changes!